### PR TITLE
docs: add info about the new profile setup

### LINKED ISSUE #342: Document about change in loading Profile
Document things changed from https://github.com/saadpasta/developerFolio/pull/333

### PR DESCRIPTION
**Pull request by naveen521kk**
about #333
fixes #342
encourage using Github Actions more about #226 because that would simply things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,8 +185,11 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
-#### Configuring GitHub Actions
-- Using the Personal Access Token you placed in the `.env` file earlier create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `OPEN_SOURCE_TOKEN` where the value matches the token value from the `.env` file in your local workspace.
+#### Configuring GitHub Actions (Recommended)
+First you should enable, Github Actions for the repository you use.
+
+Their is a [CRON setup](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) up so that it deploys your site every week, so that once you update your profile on Github it is shown on your portfolio. You can also trigger it manually using `workflow_dispatch` event, see https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch on how to do that.
```
Comment (kartikcho) [inline README.md:191]: Change this to start with: 'A configurable [CRON Job]() is setup which deploys your site....'
Maybe move the linked blog as a link like '...see [this guide]() on how to do that'

**Review by kartikcho — CHANGES_REQUESTED** [review]
I think our docs have come a long way since we started and are really descriptive now 🚀 

Need to address the minor change and we're good to go.

Also, the Vercel Preview deployments logs say it's missing env variables.
@saadpasta have you set them since this change was added and does this only affect preview deps?

**Review by kartikcho — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -184,8 +185,11 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
-#### Configuring GitHub Actions
-- Using the Personal Access Token you placed in the `.env` file earlier create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `OPEN_SOURCE_TOKEN` where the value matches the token value from the `.env` file in your local workspace.
+#### Configuring GitHub Actions (Recommended)
+First you should enable, Github Actions for the repository you use.
+
+A configurable [CRON Job](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) is setup which deploys your site every week, so that once you update your profile on Github it is shown on your portfolio. You can also trigger it manually using `workflow_dispatch` event, see [this guide](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch) on how to do that.
+
```
Comment (saadpasta) [inline README.md:192]: I think we should also add why we need to deploy the website every week or when should the user trigger the deployment manually it's not that much clear

**Comment by saadpasta** [comment]
@kartikcho env added on vercel and preview for deployment is also fixed I don't need to approve it every time I guess (not sure about this but let see 🤞 )

**Comment by saadpasta** [comment]
In other news guys @naveen521kk  @kartikcho. 

Clever Programmer the guy with almost 1 Million subscribers using DeveloperFolio to teach his students about React and JavaScript in one of his free courses. 

Video: https://www.youtube.com/watch?v=86F11Y56wmI
Clever Programmer Repo: https://github.com/CleverProgrammers/portfolio-react-cp

**Comment by saadpasta** [comment]
All props to you guys for maintaining the project so well. Thanks guys ❤️

**Comment by kartikcho** [comment]
Wow, this is huge 🎉 
Happy to know this project is gaining more and more recognition!

**Comment by naveen521kk** [comment]
@saadpasta is it fine to merge this?

**Review by saadpasta — APPROVED** [review]
LGTM 👍

**Pull request merged**